vultr/resource_block_storage.go

```diff
@@ -0,0 +1,169 @@
+package vultr
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceBlockStorage() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBlockStorageCreate,
+		Read:   resourceBlockStorageRead,
+		Update: resourceBlockStorageUpdate,
+		Delete: resourceBlockStorageDelete,
+
+		Schema: map[string]*schema.Schema{
+			"subid": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"label": {
```

Comment:
In order to be consisitent with the other resources in this provider (e.g. instance) could we rename this from “label” to “name”? I know that technically the Vultr API refers to this as a label, but for all intents and purposes it’s a name and I think it is clearer to users as well

Reply:
I've named parameters following what comes from API but I agree with you.